kotlin/scene-layer/README.md

```diff
@@ -31,8 +29,7 @@ The scene launches with a northward view of the city of Brest, France. A 3D scen
 * ArcGISSceneLayer
 * ArcGISTiledElevationSource
```

Comment:
`ArcGISTiledElevationSource` doesn't seem to be referenced here?

Reply:
ah yes, thanks: in the common design there is a Surface added which would use this class. But in our Android implementation we don't use a surface. I removed the references to that but forgot about this class! I'll update the Java one too.